src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -136,7 +136,7 @@ const ArticlePage = ({ pageData }) => {
     timestamp: props =>
       hasByline ? null : <Timestamp {...props} popOut={false} />,
     social: SocialEmbedContainer,
-    embed: props => <UnsupportedEmbed {...props} />,
+    embed: UnsupportedEmbed,
```

Comment:
Nice 👍 Should we apply this approach to the other embed types below for consistency?

Reply:
Oh right, yes have done for embeds. I didn't realise these were unnecessary.

Reply:
Yea its a bit of a weird syntax. The `props` will just get automatically passed through to the component when defined this way. Only when you need to pass in something specific or extra, like `popOut={false}` above, then you'd need to specify the overall `props` to spread in.